tools/circt-opt/circt-opt.cpp

```diff
@@ -5,17 +5,19 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "circt/Conversion/AffineToHandshake/AffineToHandshake.h"
 #include "circt/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.h"
 #include "circt/Conversion/LLHDToLLVM/LLHDToLLVM.h"
 #include "circt/Conversion/StandardToHandshake/StandardToHandshake.h"
 #include "circt/Conversion/StandardToStaticLogic/StandardToStaticLogic.h"
 #include "circt/Dialect/FIRRTL/Dialect.h"
 #include "circt/Dialect/Handshake/HandshakeOps.h"
-#include "circt/Dialect/StaticLogic/StaticLogic.h"
```

Comment:
unintended?

Reply:
It seems to be a result of `clang-format` :)